src/utils/useDidMount.ts

```diff
@@ -0,0 +1,9 @@
+import { useEffect, useRef } from 'react';
+
+export const useDidMount = () => {
+  const didMount = useRef(false);
+  useEffect(() => {
+    didMount.current = true;
+  }, []);
+  return didMount.current;
```

Comment:
I don't like useDidMount as a React practice. Specifically, this is violating the React rule: We shouldn't read a ref value in render.
Maybe, it was implemented so previously. This reveals it then.

Reply:
I will check if I can work around it, so I don't have to use useDidMount. 👍